static_report/src/utils/transformers.tsx

```diff
@@ -159,3 +260,108 @@ export function getIconForColumnType(columnDatum: ColumnSchema | undefined): {
   }
   return { backgroundColor: 'gray.500', icon: BiQuestionMark };
 }
+
+export type MetricNameMetakeyList = [MetricMetaKeys, string][];
+/**
+  Conditional scenarios:
+  
+  1. base-only (% + count) <<<
+  2. base+target (count + count)
+  3. base||target (count || count)
+  
+ * gets the list of metrics to display, based on metakey
+ */
+export function transformSRMetricsInfoList(
+  metricsList: MetricNameMetakeyList,
+  columnDatum?: ColumnSchema,
+): MetricsInfoProps[] {
+  if (!columnDatum) return [];
+  return metricsList.map(([metakey, name]) => {
+    const count = Number(columnDatum[metakey as string]);
+    const percent = count / Number(columnDatum.total);
+
+    return {
+      name,
+      metakey,
+      firstSlot: isNaN(count) ? NO_VALUE : formatIntervalMinMax(percent),
+      secondSlot: isNaN(count) ? NO_VALUE : formatAsAbbreviatedNumber(count),
+      tooltipValues: { firstSlot: formatNumber(count) },
+    };
+  });
+}
+/**
+  Conditional scenarios:
+  
+  1. base-only (% + count)
+  2. base+target (count + count) <<<
+  3. base||target (count || count) <<<
+  
+ * gets the list of metrics to display, based on metakey
+ */
+export function transformCRMetricsInfoList(
+  metricsList: MetricNameMetakeyList,
+  baseColumnDatum?: ColumnSchema,
+  targetColumnDatum?: ColumnSchema,
+  valueFormat: 'count' | 'percent' = 'percent',
```

Comment:
Is this is a optional prop?

Reply:
Nope, it's not optional, hence the default value `=`. 
Making it `?` optional and setting a default value both together would upset eslint rule :P

Reply:
Actually, correction on my last point. It's another way to supply it as optional.